maxfw/tests/test_image_utils_new.py

```diff
@@ -0,0 +1,158 @@
+# Standard libs
```

Comment:
Added a test file for the new image utils. Will have to rename the file afterwards.